apps/ejabberd/src/ejabberd_session.erl

```diff
@@ -0,0 +1,15 @@
+-module(ejabberd_session).
```

Comment:
How about naming this module `mongoose_session` as this is completely new module?